src/conventional/config.rs

```diff
@@ -135,51 +143,61 @@ fn default_types() -> Vec<Type> {
     vec![
         Type {
             r#type: "feat".into(),
+            increment: Increment::Minor,
             section: "Features".into(),
             hidden: false,
         },
         Type {
             r#type: "fix".into(),
+            increment: Increment::Patch,
             section: "Fixes".into(),
             hidden: false,
         },
         Type {
             r#type: "build".into(),
+            increment: Increment::Patch,
```

Comment:
These none fix/feature types should default to the `Increment::None`. Otherwise this is a breaking change.

Reply:
I see what you are saying and will update.  Theoretically though, should any new release, even if the interim commits are `chore` or `build` or something else minor, not expect at least a patch increment?

Reply:
> Additional types are not mandated by the Conventional Commits specification, and have no implicit effect in Semantic Versioning (unless they include a BREAKING CHANGE). 

Unless they contain a breaking change there should not be an implicit effect. If it is configurable it becomes an explicit effect. For example a `style: reformatted with new rules` should not require a new release imo.

Reply:
I see that you are following the spec.  But I feel there is a big difference between "should not" and "can not".  What if this was a continuous deployment scenario and every merge to main gets released and hence a version number?  Regardless, this PR has this covered.